{"type": "indicator", "spec_version": "2.1", "id": "indicator--c9836ce5-3288-4dfb-82db-56f6747e91b1", "created_by_ref": "identity--748e6444-f073-4c50-b558-f49be8897a81", "created": "2021-06-11T16:15:00.000Z", "modified": "2021-06-24T19:58:00.000Z", "name": "CVE-2021-28210", "description": "An unlimited recursion in DxeCore in EDK II.", "indicator_types": ["compromised"], "pattern": "[(software:cpe='cpe:2.3:a:tianocore:edk2:*:*:*:*:*:*:*:*')]", "pattern_type": "stix", "pattern_version": "2.1", "valid_from": "2021-06-11T16:15:00Z", "external_references": [{"source_name": "cve2stix", "description": "This object was created using cve2stix from the Signals Corps.", "url": "https://github.com/signalscorps/cve2stix"}], "object_marking_refs": ["marking-definition--613f2e26-407d-48c7-9eca-b8e91df99dc9"], "extensions": {"extension-definition--b463c449-d022-48b7-b464-3e9c7ec5cf16": {"extension_type": "property-extension", "all_cpe23uris": ["cpe:2.3:a:tianocore:edk2:*:*:*:*:*:*:*:*"], "vulnerable_cpeid": ["cpe:2.3:a:tianocore:edk2:*:*:*:*:*:*:*:*"]}}}